markdown/ch7.markdown

```diff
@@ -95,11 +96,32 @@ whether jiggling it in one direction or another reduces their
 dissimilarity.  By doing this repeatedly, we can try to find the
 correct alignment.
 
+You'll remember our astronaut - Eileen Collins - from chapter 3.
```

Comment:
Aus has a national English style guide, which I read as an undergrad Arts student... (I know, nerd, right?). Obviously there are exceptions for each publication, but there is a basic set of grammar rules which tends not to vary. Leaving this for now. This is the kind of thing a final O'Reilly polishing editor might tell us about. They might have such a guide.